I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/boss_scene.rpy", line 21, in script call
    call mc_attack
  File "game/boss_scene.rpy", line 42, in script
    call screen mc_attackchoice
ScriptError: could not find label 'mc_ChooseCharater'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\USER\Desktop\renpy-8.3.2-sdk.7z\renpy-8.3.2-sdk\renpy\bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "C:\Users\USER\Desktop\renpy-8.3.2-sdk.7z\renpy-8.3.2-sdk\renpy\main.py", line 652, in main
    run(restart)
  File "C:\Users\USER\Desktop\renpy-8.3.2-sdk.7z\renpy-8.3.2-sdk\renpy\main.py", line 148, in run
    renpy.execution.run_context(True)
  File "C:\Users\USER\Desktop\renpy-8.3.2-sdk.7z\renpy-8.3.2-sdk\renpy\execution.py", line 958, in run_context
    context.run()
  File "game/boss_scene.rpy", line 21, in script call
    call mc_attack
  File "game/boss_scene.rpy", line 42, in script
    call screen mc_attackchoice
  File "C:\Users\USER\Desktop\renpy-8.3.2-sdk.7z\renpy-8.3.2-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'mc_ChooseCharater'.

Windows-10-10.0.22621 AMD64
Ren'Py 8.3.2.24090902
not_yet_setted 1.0
Wed Nov 27 00:03:55 2024
